app/src/main/java/vn/sun/music_68/ui/nowplaying/FragmentPlayingList.java

```diff
@@ -0,0 +1,27 @@
+package vn.sun.music_68.ui.nowplaying;
+
+import android.os.Bundle;
+import android.view.View;
+
+import androidx.fragment.app.Fragment;
+
+import vn.sun.music_68.R;
+import vn.sun.music_68.base.BaseFragment;
+
+public class FragmentPlayingList extends BaseFragment {
```

Comment:
```suggestion
public class FragmentPlaying extends BaseFragment {
```
FragmentPlayingSongs or FragmentPlayingTracks